### PR TITLE
Vine Outfit for your Space Waifus

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
:cl: Cobby
add: Show off your botany prowess [and some skin!] with the new vine outfit, craftable with 5 ambrosia.
/:cl:

You can probably tell from the images who this outfit is loosely based off of. Like the inspiration, this is meant to be a promiscuous outfit [well as much as you can get in a <32x32 sprite without just running around nude].

The images shows the outfit with a vulgaris branch on her head, the same kind used to make the outfit. Apologies, I was trying to get the character cosplay right that I forgot the long hair covers almost the entire backside.

front side back

[For those who don't get the inspiration](https://youtu.be/sUgUMY-C7MY?t=129)

Why: Not only is this a cosmetic I hope people will enjoy, but unlike most if not all other clothing, it's pretty easily craftable. Also your 2d waifu will look FAB in it! Who DOESN'T want a sexy vine outfit anyways?!!

### CODE REVIEW

**Comment by ShizCalev** [comment]
>see it
>instantly cringe thinking of that movie

**Comment by obscolene** [comment]
disgusting codersprites

**Comment by WJohn** [comment]
but why tho

**Comment by ExcessiveUseOfCobblestone** [comment]
!!!

**Comment by bgobandit** [comment]
>tentacles

**Comment by ShadowDeath6** [comment]
I prefer the large ape suit she's wearing in the clip to be added.

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -594,6 +594,13 @@
 	item_color = "villain"
 	can_adjust = 0
 
+/obj/item/clothing/under/vine
+	name = "dress of vines"
```
Comment (Supermichael777) [inline code/modules/clothing/under/miscellaneous.dm]: based on the sprite I would go with 'botanical halter top'

desc = "Any dates a good date after the contact high"

**Comment by FantasticFwoosh** [comment]
The back has more leafy detail, doesn't look leafy enough.

**Comment by Repukan** [comment]
At least base it off the samurai Jack leaf suit

**Comment by Armhulen** [comment]
Yeah go samurai jack instead of poison ivy

**Comment by ma44** [comment]
👎 ❌ 💯 💯 💯

**Comment by Ausops** [comment]
>nobody wearing nvgs and a tacticool jumpsuit and balaclava on the edge of the screenshot

**Comment by Bawhoppen** [comment]
I like this but it could do with a better sprite

**Comment by Bawhoppen** [comment]
also why are you calling this a vine outfit when it's not made of vines (sexy legit vine outfit coming later with kudzu)

**Comment by ExcessiveUseOfCobblestone** [comment]
there are no obtainable vines, as kudzu only dispenses seeds.

But if you got a better sprite then ill let you take over. God Speed!

**Pull request closed without merging**